examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1314,6 +1315,89 @@ class GLTFTextureWebPExtension {
 
 }
 
+/**
+ * AVIF Texture Extension
+ *
+ * Specification: https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Vendor/EXT_texture_avif
+ */
+class GLTFTextureAVIFExtension {
```

Comment:
You also need this in the ctor of the loader:
```js
this.register( function ( parser ) {

    return new GLTFTextureAVIFExtension( parser );

} );
```

Reply:
whoops. missed that one.
I've pushed it now